test/globalconstraintregistrar.js

```diff
@@ -20,22 +20,24 @@ const setupGlobalConstraintRegistrarParams = async function(
                                             globalConstraintRegistrar,
```

Comment:
Let's make it just "constraint registrar (with all the necessary changes).

Reply:
this comment is not clear to me